forecast/paper/paper.tex

```diff
@@ -192,7 +206,10 @@
 Our take on this problem is in line with much of this literature; however, in
 order to avoid making any explicit assumptions, we do not attempt to make formal 
 significance statements, and instead, broadly examine the stability of our
-conclusions with respect to numerous modes of analysis.   
+conclusions with respect to numerous modes of analysis.
+% Addition upon review
+We have included a section in the Supplement that examines two assumption-lean
```

Comment:
```suggestion
In the Supplement we examine two assumption-lean
```

This is extremely nitpicky, but it feels a little weird that this is the only sentence is past tense (vs present tense).  (I understand it's because we added it after the initial review, but for a reader without context that this was added after review, the tense switch might feel weird)